apps/mobile/src/libs/components/button/button.tsx

```diff
@@ -1,19 +1,102 @@
 import React from "react";
-import { Pressable } from "react-native";
+import { Pressable, View } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
 
 import { Text } from "~/libs/components/components";
+import { BaseColor, GradientColor } from "~/libs/enums/enums";
 
+import { GradientText } from "../gradient-text/gradient-text";
 import { styles } from "./styles";
 
 type Properties = {
+	appearance?: "filled" | "outlined";
+	borderRadius: number;
+	disabled?: boolean | null;
 	label: string;
 	onPress: () => void;
 };
 
-const Button: React.FC<Properties> = ({ label, onPress }) => {
+const Button: React.FC<Properties> = ({
+	appearance = "filled",
+	borderRadius,
+	disabled,
+	label,
+	onPress,
+}) => {
+	const FIRST_COLOR_STOP = -0.4;
+	const SECOND_COLOR_STOP = 0.9;
+	const IS_FILLED = appearance === "filled";
+
 	return (
-		<Pressable onPress={onPress} style={styles.btn}>
-			<Text style={styles.label}>{label}</Text>
+		<Pressable disabled={disabled} onPress={onPress} style={[styles.wrapper]}>
+			{({ pressed }) =>
+				pressed ? (
+					<LinearGradient
```

Comment:
I believe we can leave just `LinearGradient ` as a wrapper and on pressing it just change the `colors` set. (For example, by default it will be [BaseColor.BLACK, BaseColor.Black] and when pressed - [...GradientColor.BLUE])